storybook/pages/AirdropsSettingsPanelPage.qml

```diff
@@ -250,6 +253,19 @@ SplitView {
 
                 text: "Owner and TMaster tokens deployed"
             }
+
+            Row {
+                Label {
+                    anchors.verticalCenter: parent.verticalCenter
+                    text: "Assets Decimals"
+                }
+
+                TextField {
+                    id: decimalsText
+                    text: "2"
```

Comment:
I believe it might be better to have a constant for this? It is also repeated in L183 of storybook/pages/HoldingsDropdownPage.qml wdyt?

Reply:
I think it's a common approach for Storybook to hardcode test data into "<something>Page.qml".

Here is an example of its use:

<img width="1562" alt="image" src="https://github.com/status-im/status-desktop/assets/1083341/7a9339b9-c8ce-443f-bea2-4d7f0652ec2e">